vtex-apps/store-components
# Fix/socialshare account

#### What problem is this solving?

The account name is being displayed as default in social share text instead of the store name

#### How to test it?

Go to any PDP and click to share on any social media, you will see that by default it's displaying the account name instead of the store name

[Before](https://www.rttshop.com.br/sc2000-verde-1kg/p)

<img width="751" alt="image" src="https://github.com/vtex-apps/store-components/assets/13649073/abe8aa7d-bcca-4873-bfa6-cc41ed3b89c0">

[After](https://iespinoza--b2brematiptop.myvtex.com/sc2000-verde-1kg/p)

<img width="748" alt="image" src="https://github.com/vtex-apps/store-components/assets/13649073/553f9916-64b7-4d7f-9de1-21d1b3b67991">

#### Related to / Depends on

Zendesk: #834432

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/muF624U0m4fBFuIV4z/giphy.gif)
